lib/ansible/modules/hostname.py

```diff
@@ -29,8 +29,8 @@
     use:
         description:
             - Which strategy to use to update the hostname.
-            - If not set we try to autodetect, but this can be problematic, specially with containers as they can present misleading information.
-        choices: ['generic', 'debian','sles', 'redhat', 'alpine', 'systemd', 'openrc', 'openbsd', 'solaris', 'freebsd']
+            - If not set we try to autodetect, but this can be problematic, especially with containers as they can present misleading information.
```

Comment:
specially is correct in this context

Reply:
[Wiktionary](https://en.wiktionary.org/wiki/specially) lists that usage as non-standard: "Senses 2 and 3 are non-standard; in such cases, especially is far more common, and considered correct."

I'll avoid the bikeshed and change it to "particularly" though :)